web/views/api.py

```diff
@@ -15,9 +16,10 @@
 
 from stronghold.decorators import public
 
+from core.importer.contract_exporter import ContractExporter
 from core.importer.datasets_exporter import DatasetsExporter
 from core.importer.projects_exporter import ProjectsExporter
-from core.models import User, Cohort, Dataset, Partner, Project, DiseaseTerm
+from core.models import User, Cohort, Dataset, Partner, Project, DiseaseTerm, Contract
```

Comment:
`Contract` seems not to be used anywhere

Reply:
fixed